watsononlinestore/watson_online_store.py

```diff
@@ -267,6 +267,19 @@ def get_discovery_response(self, input_text):
             # This dumps a ton of results for us to peruse:
             pprint(discovery_response)
 
+        filtered_results = []
```

Comment:
we don't ever use "num_matches" right? i think this can be shorter -- rename discovery_response to resp to make it pep8 friendly:

if 'results' in resp:
  resp[:] = [x for x in resp if 'score' and x['score']>=0.25 ]